doc/modules/preprocessing_targets.rst

```diff
@@ -7,6 +6,73 @@
 Transforming the prediction target (``y``)
 ==========================================
 
+Transforming target in regression
+---------------------------------
+
+:class:`TransformTargetRegressor` transforms the targets ``y`` before fitting a
```

Comment:
Does anyone else think this would be better as `TransformedTargetRegressor`?

Reply:
I agree. I like `TransformedTargetRegressor` better as well.